### DISCUSSION TITLE
Add "No Icon Set" Icon for slots

### DISCUSSION THREAD

**Discussion by Arcadius2006** [LBPUnion/ProjectLighthouse#812] [Ideas]
instead of displaying the question mark, make levels without an icon display a unique graphic, maybe reminiscent of the actual one
 Example Image: 
 ![emptylevelbadge](https://github.com/LBPUnion/ProjectLighthouse/assets/93880348/c172060f-e959-4ead-ab21-7d1d19ccf1dd)

**Comment by sudokoko** [comment]
Lighthouse has a configuration option for this. I'm open to replacing the question mark however this will be used across the entire website when an asset doesn't exist. Is this intended?

**Comment by Arcadius2006** [comment]
no no i want it to be specifically for levels with no icon set, levels that use an ingame icon or error out in some other way will still use the question mark, possibly the question mark graphic could be also touched up though

**Reply by sudokoko** [reply]
Not sure if there is a way to discern the difference between the two. If you know of a way feel free to let us know

**Reply by Arcadius2006** [reply]
levels with icons should always have either a hash or a guid of an icon attached to them, if the guid points to the "no icon" graphic (or if it doesnt point to anything), it should show the no icon graphic lighthouse has

**Reply by sudokoko** [reply]
I believe if it returns a GUID then it'll show the no icon hash. I'll look into this.
